python/src/MetropolisHastingsImplementation_doc.i.in

```diff
@@ -0,0 +1,499 @@
+    If provided, its input dimension must match the chain dimention and its
+    output dimension must match the conditional distribution parameter dimension.
+covariates : 2-d sequence of float, optional
+    Parameters of the linkFunction for each observation.
+    If provided, its dimension must match the parameter dimension of the linkFunction."
+%enddef
+%feature("docstring") OT::MetropolisHastingsImplementation::setLikelihood
+OT_MetropolisHastings_setLikelihood_doc
+
+// ---------------------------------------------------------------------
+
+%define OT_MetropolisHastings_computeLogLikelihood_doc
+"Compute the logarithm of the likelihood w.r.t. observations.
+
+Parameters
+----------
+currentState : sequence of float
+    Current state.
+
+Returns
+-------
+logLikelihood : float
+    Logarithm of the likelihood w.r.t. observations
+    :math:`(\vect{y}^1, \dots, \vect{y}^n)`."
+%enddef
+%feature("docstring") OT::MetropolisHastingsImplementation::computeLogLikelihood
+OT_MetropolisHastings_computeLogLikelihood_doc
+
+// ---------------------------------------------------------------------
+
+%define OT_MetropolisHastings_computeLogPosterior_doc
+"Compute the logarithm of the posterior.
+
+Parameters
+----------
+currentState : sequence of float
+    Current state.
+
+Returns
+-------
+logLikelihood : float
+    Logarithm of the posterior
+    :math:`(\vect{y}^1, \dots, \vect{y}^n)`."
+%enddef
+%feature("docstring") OT::MetropolisHastingsImplementation::computeLogPosterior
+OT_MetropolisHastings_computeLogPosterior_doc
+
+// ---------------------------------------------------------------------
+
+%define OT_MetropolisHastings_getBurnIn_doc
+"Get the length of the burn-in period.
+
+Returns
+-------
+lenght : int
+    Length of the burn-in period, that is the number of first iterates of the
+    MCMC chain which will be thrown away when generating the sample."
+%enddef
+%feature("docstring") OT::MetropolisHastingsImplementation::getBurnIn
+OT_MetropolisHastings_getBurnIn_doc
+
+// ---------------------------------------------------------------------
+
+%define OT_MetropolisHastings_setBurnIn_doc
+"Set the length of the burn-in period.
+
+Parameters
+----------
+lenght : int
+    Length of the burn-in period, that is the number of first iterates of the
+    MCMC chain which will be thrown away when generating the sample."
+%enddef
+%feature("docstring") OT::MetropolisHastingsImplementation::setBurnIn
+OT_MetropolisHastings_setBurnIn_doc
+
+// ---------------------------------------------------------------------
+
+%define OT_MetropolisHastings_getConditional_doc
+"Get the *conditional* distribution.
+
+Returns
+-------
+conditional : :class:`~openturns.Distribution`
+    Distribution taken into account in the definition of the likelihood, whose
+    PDF with parameters :math:`\vect{w}` corresponds to :math:`f(.|\vect{w})`
+    in the equations of the
+    :ref:`target distribution's PDF <PDF_target_formula>`."
```

Comment:
Apparently, you removed `PDF_target_formula` (which is why Sphinx issues a warning), but I am not sure why. It is still relevant if you do not specify a `linkFunction` in `setLikelihood`.

Reply:
yes, maybe its better to keep the general one and detail that in setllikelihood